### DISCUSSION TITLE
Where to index historical deployments?

### DISCUSSION THREAD

**Discussion by PaulRBerg** [sablier-labs/v2-docs#97] [General]
The [deployment addresses](https://docs.sablier.com/contracts/v2/deployments) will change when we deploy V2.1.

Where should we keep the reference for V2.0?

**Comment by PaulRBerg** [comment]
@andreivladbrg I would you like to have a think about this problem given that we are getting closer to deploying Sablier V2.1

**Comment by andreivladbrg** [comment]
My simple answer is: Still on the docs page. By default, we show the 2.1 contracts, but we will add a line that says something like, "For older versions of the contracts, click here." This will redirect to the actual [deployment addresses](https://docs.sablier.com/contracts/v2/deployments) page.

Also, we can also link the [v2-deployments](https://github.com/sablier-labs/v2-deployments/).

**Reply by PaulRBerg** [reply]
> to the actual deployment addresses

What does "actual deployment addresses" mean?

> we can also link the v2-deployments

That repo does not contain addresses.

**Reply by andreivladbrg** [reply]
> What does "actual deployment addresses" mean?

actual deployment addresses(the ones we have shown at the moment) = 2.0

When we deploy 2.1, that would be the default. And we will have a redirect to, at that point, the old version (2.0).

> That repo does not contain addresses

It does if you know where to look: https://github.com/sablier-labs/v2-deployments/blob/3cf4faaa8787d837016fdbf589e391e705a4a84f/core/v1.0.0/broadcasts/mainnet.json#L67

**Reply by PaulRBerg** [reply]
> And we will have a redirect to, at that point, the old version (2.0).

What exactly would we redirect to?

> It does if you know where to look

Ah, yeah. But that is not a user-friendly interface for viewing addresses. We need a website not a JSON file.

**Reply by andreivladbrg** [reply]
> What exactly would we redirect to?

Something like this: 
https://docs.sablier.com/contracts/v2/deployments/older-2.0

**Reply by PaulRBerg** [reply]
Ok - so your suggestion is to:

1. Maintain the previous in a document that is by default hidden (not included in the sidebar)
2. Redirect to the said document in the default deployment page

**Comment by PaulRBerg** [comment]
Closing - this is how we resolved this:

- The [default deployments page](https://docs.sablier.com/contracts/v2/deployments) points to the latest protocol version (V2.1 at the time of posting this comment)
- There's a new section in the sidebar called "Older Deployments", which contains pages for other deployments (e.g., V2.0)

<img width="300" alt="SCR-20231230-luzq" src="https://github.com/sablier-labs/v2-docs/assets/8782666/dc2f359a-2f5b-4fdf-90f7-7c4049ed7d31">
